src/app/app.module.ts

```diff
@@ -3,22 +3,39 @@ import { BrowserModule } from '@angular/platform-browser'
 import { HttpClient, HttpClientModule } from '@angular/common/http'
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 
-import { TranslateModule, TranslateLoader } from '@ngx-translate/core'
+import { TranslateModule } from '@ngx-translate/core'
+import { TranslateLoader } from '@ngx-translate/core';
 import { TranslateHttpLoader } from '@ngx-translate/http-loader'
+import { FormsModule } from '@angular/forms';
+import { ReactiveFormsModule } from '@angular/forms';
+
+import { MatAutocompleteModule } from '@angular/material/autocomplete';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatInputModule } from '@angular/material/input'
+import { MatIconModule } from '@angular/material/icon';
+import { MatButtonModule } from '@angular/material/button';
+
 import { AppComponent } from './app.component';
+import { PatientSearchComponent } from './patient-search/patient-search.component';
+import { PatientService } from './services/patient-service';
```

Comment:
please organise imports in 3 blocks, 1st angular 2nd external libraries and 3rd our application imports

Reply:
done